docs/operator.rst

```diff
@@ -476,11 +476,10 @@ See also `join`_, `cross`_, `spread`_ and `phase`_.
 concat
 ------
 
-The ``concat`` operator allows you to `concatenate` the items emitted by two or more channels to a new channel, in such
-a way that the items emitted by the resulting channel are in same order as they were when specified as operator arguments.
+The ``concat`` operator allows you to `concatenate` the items emitted by two or more channels to a new channel. The items emitted by the resulting channel are in the same order as specified in the operator arguments.
 
-In other words it guarantees that given any `n` channels, the concatenation channel emits the items proceeding from the channel `i+1 th`
-only after `all` the items proceeding from the channel `i th` were emitted.
+Given ``n`` number of channels, the concatenation channel emits the items proceeding from the ``(i+1)`` th channel 
+only after `all` the items proceeding from the ``i`` th channel were emitted.
```

Comment:
```suggestion
Given `n` number of channels, the concatenation channel emits the items proceeding from the `i+1 th` channel 
only after `all` the items proceeding from the `i th` channel were emitted.
```

Note single ` wrap the text as italic, and `` as code

Reply:
All noted. Thanks.